### PR TITLE
N22: Release

### PR DESCRIPTION
**Pull request by AngelOnFira**
Part of #640

Preview: https://agitated-saha-af60ca.netlify.app/news/022/

Todo:
- [x] Follow the plan
- [x] Grammarly pass
- [ ] Final review with everyone

### CODE REVIEW

File: content/news/022/index.md [hunk 1]
```diff
@@ -412,6 +412,31 @@ rusty-editor updates:
 [rg3d_twitter]: https://twitter.com/DmitryNStepanov
 [rg3d_terrain_reddit]: https://www.reddit.com/r/rust/comments/nlnfdb/timelapse_of_terrain_editing_in_rustyeditor_which/
 
+### [Nestadia][nestadia]
+
+![Zelda running on Nestadia](nestadia.png)
+
+[Nestadia][nestadia] by [@zer0x64], [@junior-n30] and [@CBenoit] is a
```
Comment (17cupsofcoffee) [inline content/news/022/index.md:419]: These @ links are broken - should probably just be text.

Same on the Rustcraft section.

**Review by 17cupsofcoffee — COMMENTED** [review]
One minor issue with some links:

**Comment by 17cupsofcoffee** [comment]
Also, don't forget to remove the empty sections from the end of the newsletter and the TOC if there's no content to put in them:

* Popular Workgroup Issues in Github
* Meeting Minutes
* Requests for Contribution (might be able to copy some of the recurring links from previous newsletters instead of deleting this)
* Jobs
* Bonus

**Review by ozkriff — COMMENTED** [review]
> https://www.reddit.com/r/rust_gamedev/comments/nhdwky/i_used_rust_to_make_an_mmo_in_48_hours_for/?utm_source=share&utm_medium=web2x&context=3

this link can be stripped-down of its arguments

**Comment by kvark** [comment]
The preview doesn't how any images?

**Comment by AngelOnFira** [comment]
> The preview doesn't how any images?

I'll take a look, I'm sure I messed something up :sweat_smile:

**Review by kvark — APPROVED** [review]
checked the structure and skimmed the grammar, looks good

**Review by 17cupsofcoffee — APPROVED** [review]
Also looks good to me :+1:

**Review by ozkriff — APPROVED** [review]
lgtm! :ship:it!

**Pull request merged**